src/main/java/seedu/address/storage/JsonSerializableNetConnect.java

```diff
@@ -20,6 +20,7 @@
 class JsonSerializableNetConnect {
 
     public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
+    public static final String MESSAGE_DUPLICATE_ID = "Persons list contains duplicate id(s).";
```

Comment:
Perhaps can standardise to write IDs as ID or id or Id in strings

Reply:
I think I standardised to `id` for messages! Some places use `ID` as it is used in parameters, similar to `n/NAME` so `i/ID`. Hopefully it doesn't differ anywhere else hehe